Misc/NEWS.d/next/Core_and_Builtins/2024-09-06-19-23-44.gh-issue-120221.giJEDT.rst

```diff
@@ -0,0 +1,2 @@
+asyncio REPL is now again properly recognizing KeyboardInterrupts. Display
```

Comment:
```suggestion
asyncio REPL is now again properly recognizing :exc:`KeyboardInterrupt`. Display
```

Reply:
That's not the same word. Let's do stuff like that in a subsequent PR please, so we don't waste time on CI now.

Reply:
Agreed. The upcoming release is more important now.

Reply:
FTR, to put the s, you would do <code>:exc:\`KeyboardInterrupt\`\s</code>